### PR TITLE
feat: add options goToAndPlay and goToAndStop

### PR DESCRIPTION
**Pull request by grzegorz-s**
I created two wrapper functions for goToAndPlay and goToAndStop.
I had a problem to run test even before changes. Is that known issue?

### CODE REVIEW

**Comment by GabrielFn** [comment]
I really wanted this feature

**Comment by antonbks** [comment]
Can this be merged please

**Comment by sultanmyrza** [comment]
I also need this feature please merge to master

**Comment by Zelda256** [comment]
I really need this feature, please merge

**Review by mfilype — APPROVED** [review]

**Comment by Namzakku** [comment]
please merge this feature

File: src/stories/lottie-goToAndPlay.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+export default class LottieGoToAndPlay extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isStopped: true,
+      value: 0,
+      isFrame: false,
+      loop: false,
```
Comment (deshario9) [inline src/stories/lottie-goToAndPlay.js:14]: .

**Pull request closed without merging**